test/integration/resourcemanager/health/health_test.go

```diff
@@ -834,6 +923,43 @@ func generateAlertmanagerTestResource(name string) *monitoringv1.Alertmanager {
 	}
 }
 
+func generateCertificateTestResource(name string) *certv1alpha1.Certificate {
+	return &certv1alpha1.Certificate{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: testNamespace.Name,
```

Comment:
Don't we also want to set the generation to 42 here similar to other `generate*` functions?

Reply:
We can but we don't really need it FMPOV. You can't set the generation when creating a resource with a real client. I could rather check if we can get rid of the others.